### PR TITLE
feat: LT-2032 Exhibition Details

### PR DESCRIPTION
**Pull request by rorysproule1**

### CODE REVIEW

File: cases/services.py [hunk 1]
```diff
@@ -354,7 +354,10 @@ def _generate_post_data_and_errors(keys, request_data, action):
 def _get_total_goods_value(case):
     total_value = 0
     for good in case.get("application").get("goods", []):
-        total_value += Decimal(good["value"]).quantize(Decimal(".01"))
+        if good.get("value"):
+            total_value += Decimal(good["value"]).quantize(Decimal(".01"))
+        else:
+            return None
```
Comment (BenAndrew-Kainos) [inline cases/services.py]: What about if you have a mixture of goods with values & without values (i.e. some software & some physical). I'd suggest removing the else return None bit

File: core/builtins/custom_tags.py [hunk 2]
```diff
@@ -67,6 +67,13 @@ def str_date(value):
     )
 
 
+@register.filter
+@stringfilter
+def str_date_only(value):
+    return_value = do_timezone(datetime.datetime.strptime(value, DATE_NO_TIME), "Europe/London")
+    return return_value.strftime("%d %B " "%Y")
```
Comment (BenAndrew-Kainos) [inline core/builtins/custom_tags.py]: What's up with this string here (why is it two strings?)

Comment (rorysproule1) [reply core/builtins/custom_tags.py]: Corrected to one string.

File: cases/services.py (hunk 1 above)
Comment (rorysproule1) [reply cases/services.py]: Removed the else.

File: cases/services.py [hunk 3]
```diff
@@ -354,7 +354,8 @@ def _generate_post_data_and_errors(keys, request_data, action):
 def _get_total_goods_value(case):
     total_value = 0
     for good in case.get("application").get("goods", []):
-        total_value += Decimal(good["value"]).quantize(Decimal(".01"))
+        if good.get("value"):
```
Comment (stephanieboyd) [inline cases/services.py]: Comment as to why it can be zero would be good here

File: conf/constants.py [hunk 4]
```diff
@@ -1,6 +1,7 @@
 from enum import Enum
 
 ISO8601_FMT = "%Y-%m-%dT%H:%M:%S.%fZ"
+DATE_NO_TIME = "%Y-%m-%d"
```
Comment (stephanieboyd) [inline conf/constants.py]: minor - `DATE_FORMAT` might have been a better name. it's not a date, it's a format :)

File: core/builtins/custom_tags.py [hunk 5]
```diff
@@ -67,6 +67,13 @@ def str_date(value):
     )
 
 
+@register.filter
+@stringfilter
+def str_date_only(value):
+    return_value = do_timezone(datetime.datetime.strptime(value, DATE_NO_TIME), "Europe/London")
```
Comment (stephanieboyd) [inline core/builtins/custom_tags.py]: minor - return_value isn't a good name, call it what it is please :)

File: templates/case/applications/mod-clearance.html [hunk 6]
```diff
@@ -57,6 +49,39 @@
 	</div>
 
 	<div class="govuk-accordion app-accordion" data-module="govuk-accordion" id="accordion-default">
+		{% if case.case_type.sub_type.key == "exhibition_clearance"  %}
```
Comment (stephanieboyd) [inline templates/case/applications/mod-clearance.html]: Is there not a constant for these sub types? If there isn't, there probably should be as it's used a lot and in different files

File: cases/services.py (hunk 3 above)
Comment (rorysproule1) [reply cases/services.py]: Added a comment to explain.

File: conf/constants.py (hunk 4 above)
Comment (rorysproule1) [reply conf/constants.py]: Changed to DATE_FORMAT

File: core/builtins/custom_tags.py (hunk 5 above)
Comment (rorysproule1) [reply core/builtins/custom_tags.py]: Changed to date_str

File: templates/case/applications/mod-clearance.html (hunk 6 above)
Comment (rorysproule1) [reply templates/case/applications/mod-clearance.html]: Added a constant

**Pull request merged**